code/modules/reagents/chemistry/reagents/misc_reagents.dm

```diff
@@ -514,8 +514,77 @@
 	if(M.mind?.assigned_role != "Clown")
 		REMOVE_TRAIT(M, TRAIT_COMIC_SANS, id)
 		M.RemoveElement(/datum/element/waddling)
+		M.DeleteComponent(/datum/component/squeak)
```

Comment:
Looks like this might fix clowns having their squeakiness taken away when jestosterone ends, can you add that to the changelog?